AGENTS.md

```
@@ -326,168 +237,118 @@ const form = useForm({ resolver: zodResolver(schema) })
 
-[optional body]
-```
+---
+
+## 🔧 CRD Architecture (Kubernetes-inspired)
 
-**Types:** `feat | fix | docs | style | refactor | test | chore`
+### Resource Hierarchy
 
-**Examples:**
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)